integrations/philip-click-up/.port/resources/port-app-config.yaml

```diff
@@ -0,0 +1,48 @@
+createMissingRelatedEntities: true
+deleteDependentEntities: true
+resources:
+  - kind : team
+    selector:
+      query: "true"
+    port:
+      entity:
+        mappings:
+          identifier: .id
+          title: .name
+          blueprint: '"clickupTeam"'
+          properties:
+            url:  \"https://app.clickup.com/\" + .id+ \"/home\"
+            members: "[.members[].user]"
+  - kind : project
+    selector:
+      query: "true" 
+    port:
+      entity:
+        mappings:
+          identifier: .key
```

Comment:
the response from clickup does not have `key`. it is `id` instead

Reply:
Done.